src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -1,85 +1,226 @@
+        ExtractableResponse<Response> response = 지하철_노선_목록_조회_요청();
 
         // then
         // 지하철_노선_목록_응답됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+
         // 지하철_노선_목록_포함됨
+        List<Long> expectedLineIds = Stream.of(createResponse1, createResponse2)
+                .map(this::지하철_노선_응답_아이디)
+                .collect(Collectors.toList());
+        List<Long> resultLineIds = response.jsonPath()
+                .getList(".", LineResponse.class)
+                .stream()
+                .map(LineResponse::getId)
+                .collect(Collectors.toList());
+        assertThat(resultLineIds).containsAll(expectedLineIds);
     }
 
     @DisplayName("지하철 노선을 조회한다.")
     @Test
     void getLine() {
         // given
         // 지하철_노선_등록되어_있음
+        ExtractableResponse<Response> createResponse = 지하철_노선_생성_요청(params);
 
         // when
         // 지하철_노선_조회_요청
+        String uri = createResponse.header("Location");
+        ExtractableResponse<Response> response = 지하철_노선_조회_요청(uri);
 
         // then
         // 지하철_노선_응답됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertAll(
+                () -> assertThat(지하철_노선_응답_아이디(response)).isEqualTo(지하철_노선_응답_아이디(createResponse)),
+                () -> assertThat(지하철_노선_응답_이름(response)).isEqualTo(지하철_노선_응답_이름(createResponse)),
+                () -> assertThat(지하철_노선_응답_색상(response)).isEqualTo(지하철_노선_응답_색상(createResponse))
+        );
     }
 
     @DisplayName("지하철 노선을 수정한다.")
     @Test
     void updateLine() {
         // given
         // 지하철_노선_등록되어_있음
+        ExtractableResponse<Response> createResponse = 지하철_노선_생성_요청(params);
 
         // when
         // 지하철_노선_수정_요청
+        params.clear();
+        params.put("name", "3호선");
+        params.put("color", "bg-yellow-600");
+        String uri = createResponse.header("Location");
+        ExtractableResponse<Response> response = 지하철_노선_수정_요청(params, uri);
 
         // then
         // 지하철_노선_수정됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        assertAll(
+                () -> assertThat(지하철_노선_응답_아이디(response)).isEqualTo(지하철_노선_응답_아이디(createResponse)),
+                () -> assertThat(지하철_노선_응답_이름(response)).isEqualTo("3호선"),
+                () -> assertThat(지하철_노선_응답_색상(response)).isEqualTo("bg-yellow-600")
+        );
+
     }
 
     @DisplayName("지하철 노선을 제거한다.")
     @Test
     void deleteLine() {
         // given
         // 지하철_노선_등록되어_있음
+        ExtractableResponse<Response> createResponse = 지하철_노선_생성_요청(params);
 
         // when
         // 지하철_노선_제거_요청
+        String uri = createResponse.header("Location");
+        ExtractableResponse<Response> response = 지하철_노선_제거_요청(uri);
 
         // then
         // 지하철_노선_삭제됨
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+    }
+
+    private ExtractableResponse<Response> 지하철_노선_생성_요청(Map<String, String> params) {
```

Comment:
혹시 `Map`을 사용하신 별도의 이유가 있을까요?

아래와 같이 `LineRequest`를 사용하시는건 어떨까요?

```suggestion
    private ExtractableResponse<Response> 지하철_노선_생성_요청(LineRequest lineRequest) {
```